scripts/publish/cloudbuild.yaml

```
@@ -117,6 +117,24 @@ steps:
     entrypoint: "node"
     args: ["/usr/src/app/pipeline.js", "--package=firebase-tools@latest", "--publish"]
 
+  # Grab the latest version, store in workspace
+  - id: "Read New Version Number from npm"
+    name: "node"
+    entrypoint: "sh"
+    args: ["-c", "npm view firebase-tools version > /workspace/version_number.txt"]
+  # Publish the Firebase docker image
+  # Publish the Firebase docker image
+  - name: "gcr.io/cloud-builders/docker"
+    entrypoint: "sh"
+    args:
+      [
+        "-c",
+        "docker build -t gcr.io/$PROJECT_ID/firebase:$(cat /workspace/version_number.txt) -t gcr.io/$PROJECT_ID/firebase:latest -f ./firebase-docker-image/Dockerfile .",
+      ]
```

If you're gonna do linebreaks, might as well use the yaml syntax to your benefit

```suggestion
      - "-c"
      - "docker build -t gcr.io/$PROJECT_ID/firebase:$(cat /workspace/version_number.txt) -t gcr.io/$PROJECT_ID/firebase:latest -f ./firebase-docker-image/Dockerfile ."
```